common-design-patterns/site-menu.md

```diff
@@ -0,0 +1,246 @@
+---
+altLangPage: "https://conception.canada.ca/configurations-conception-communes/menu-site.html"
+breadcrumbs:
+  - title: "About Canada.ca"
+    link:  "https://www.canada.ca/en/government/about.html"
+  - title: Canada.ca design system
+    link: "https://www.canada.ca/en/government/about/design-system.html"
+  - title: Template and pattern library
+    link: "https://www.canada.ca/en/government/about/design-system/pattern-library.html"    
+  - title: Global header
+    link: "https://design.canada.ca/common-design-patterns/global-header.html"
```

Comment:
@delisma -  Breadcrumbs are generated by new config, but we don't have a good solution for children yet

Reply:
I understand what you mean. If we put all the children in one folder, we get stucked with the folder name in the url. We could namespace the files like global-header-site-menu.md, globale-header-signature.md, but we're still stucked with a namespace in the url. Unless we add a permalink property to children pages...🤔

Reply:
Ok the permalink strategy does work. You will see that I moved all the children pages in the global-header folder. Then at the _config.yml level, I set up the breadcrumbs and the permalinks for the children. So the url is still listing every component under `common-design-patterns` but the breadcrumbs have the `Global header` added to these pages.